src/main/java/org/sciborgs1155/robot/shooter/Shooter.java

```diff
@@ -0,0 +1,72 @@
+package org.sciborgs1155.robot.shooter;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.util.function.DoubleSupplier;
+import monologue.Logged;
+import monologue.Monologue.LogBoth;
+import org.sciborgs1155.robot.Robot;
+
+public class Shooter extends SubsystemBase implements Logged, AutoCloseable {
+  private final FlywheelIO flywheel;
+
+  @LogBoth
+  private final PIDController pidController =
+      new PIDController(ShooterConstants.kp, ShooterConstants.ki, ShooterConstants.kd);
```

Comment:
do a static import for the ShooterConstants (import static org.sciborgs1155.robot.ShooterConstants.*). That way, you can access all the constants without ShooterConstants., so you can just have kp and ki and kd, etc.